widget/locale_test.go

```diff
@@ -7,17 +7,15 @@ import (
 )
 
 func TestGetLocaleWeekStart(t *testing.T) {
-	assert.Equal(t, "Monday", lookupLocaleSetting(localeWeekStartKey, ""))
-	assert.Equal(t, "Monday", lookupLocaleSetting(localeWeekStartKey, "en"))
-	assert.Equal(t, "Monday", lookupLocaleSetting(localeWeekStartKey, "en-GB"))
-	assert.Equal(t, "Sunday", lookupLocaleSetting(localeWeekStartKey, "en-US"))
-	assert.Equal(t, "Sunday", lookupLocaleSetting(localeWeekStartKey, "es-US"))
+	assert.Equal(t, "Monday", lookupLocaleSetting("").weekStartDay.String())
+	assert.Equal(t, "Monday", lookupLocaleSetting("en").weekStartDay.String())
+	assert.Equal(t, "Monday", lookupLocaleSetting("en-GB").weekStartDay.String())
+	assert.Equal(t, "Sunday", lookupLocaleSetting("en-US").weekStartDay.String())
+	assert.Equal(t, "Sunday", lookupLocaleSetting("es-US").weekStartDay.String())
 }
 
 func TestGetLocaleDateFormat(t *testing.T) {
-	assert.Equal(t, "02/01/2006", lookupLocaleSetting(localeDateFormatKey, ""))
-	assert.Equal(t, "02/01/2006", lookupLocaleSetting(localeDateFormatKey, "en"))
```

Comment:
Any reason these two were removed?

Reply:
I'd have to change the expected value to empty string, or else test via the `getLocaleDateFormat` function, which uses the actual system locale. That function now handles the "" -> defaultDateFormat substitution for table entries that don't have a date format set.

Reply:
I don't understand - pasting this line in passes as expected, but is not included in date format tests whereas it still is tested in the week start test:
```go
	assert.Equal(t, "02/01/2006", lookupLocaleSetting("en").dateFormat)
```

Reply:
You are right - I added them back. Those 2 are both hitting the default entry in the map which does explicitly specify the date format.